fvcore/common/checkpoint.py

```diff
@@ -258,18 +258,26 @@ class PeriodicCheckpointer:
     multiple of period or if `max_iter` is reached.
     """
 
-    def __init__(self, checkpointer: Any, period: int, max_iter: int = None):
+    def __init__(
+        self,
+        checkpointer: Any,
+        period: int,
+        max_iter: int = None,
+        n_keep: int = None,
```

Comment:
Could you rename it to "max_to_keep"? This is how the same argument is called in `tf.train.Saver` & `tf.train.CheckpointManager` and having some consistency is nice. On the design our `Checkpointer` is like `tf.train.Checkpoint` and `PeriodicCheckpointer` is like `tf.train.CheckpointManager`.

Reply:
Done.